src/EA.Weee.Web.Tests.Unit/Areas/Aatf/Controller/ManageEvidenceNotesControllerCreateEvidenceNoteTests.cs

```diff
@@ -316,5 +317,113 @@ public async Task CreateEvidenceNotePost_GivenModelStateErrors_ErrorsShouldBeOrd
             ManageEvidenceController.ModelState.ElementAt(6).Key.Should().Be("CategoryValues2");
             ManageEvidenceController.ModelState.ElementAt(7).Key.Should().Be("CategoryValues");
         }
+
+        [Fact]
+        public async Task CreateEvidenceNoteGet_Should_CallAndClearSessionService()
+        {
+            //Act
+            await ManageEvidenceController.CreateEvidenceNote(OrganisationId, AatfId, true);
+
+            //Assert
+            A.CallTo(() => SessionService.GetTransferSessionObject<EditEvidenceNoteViewModel>(A<HttpSessionStateBase>._, A<string>._)).MustHaveHappenedOnceExactly();
+            A.CallTo(() => SessionService.SetTransferSessionObject(A<HttpSessionStateBase>._, null, A<string>._)).MustHaveHappenedOnceExactly();
```

Comment:
Rather than ignoring the parameters for GetTransferSessionObject & SetTransferSessionObject here, you want to check they are correct. Example in TransferEvidenceNotePost_GivenRequestIsCreated_SessionShouldBeUpdated.

Similar issues in the test file :)